### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by 9585999**
change watch path controllers

### CODE REVIEW

**Comment by QWp6t** [comment]
That line should be deleted entirely

File: resources/assets/config.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "cacheBusting": "[name]_[hash:8]",
   "watch": [
     "app/**/*.php",
+    "app/controllers/**/*.php",
```
Comment (kalenjohnson) [inline resources/assets/config.json]: No need for this, the line above has everything covered

Comment (9585999) [reply resources/assets/config.json]: did not dare to delete)

**Comment by kalenjohnson** [comment]
Thanks!

**Comment by 9585999** [comment]
@kalenjohnson no problem :)

**Pull request merged**